_data/simple-icons.json

```
@@ -3410,6 +3410,16 @@
             "hex": "BE2323",
             "source": "https://www.ebox.ca"
         },
+        {
+            "title": "Eclipse Adoptium",
+            "hex": "FF1464",
+            "source": "https://adoptium.net",
```

```suggestion
            "source": "https://www.eclipse.org/org/artwork/",
```
This page should remain even if the PDF is updated or removed in future and should be used as a source.